scout/build/case.py

```diff
@@ -34,6 +34,17 @@ def build_phenotype(phenotype_id, adapter):
     return phenotype
 
 
+def _populate_pipeline_info(case_obj, case_data):
```

Comment:
```suggestion
def _assign_pipeline_info(case_obj, case_data):
```
Or just `build_pipeline_info` - its just language, but we've generally called the addition of data to the in memory objects for web rendering 'population' in contrast to the 'build' for converting the objects from memory for db storage, though there are some unfortunate reuses.. 😓

Reply:
Yes the other key is already collected fine, because the case key has the same name that is taken from the config file